src/style.css

```diff
@@ -1,3 +1,82 @@
-body {
+body,
+html {
   margin: 0;
+  padding: 0;
+  font-family: 'Roboto font', sans-serif;
```

Comment:
The font-family 'Roboto font' seems incorrect. It should be 'Roboto', sans-serif. Ensure that the font is correctly loaded in your HTML or via a CSS import.